perf/benchmark/graph_plotter/graph_plotter.py

```diff
@@ -174,15 +179,20 @@ def get_parser():
     parser.add_argument(
         "--query_str",
         help="Specify the qps or conn query_str that will be used to query your y-axis data based on the CSV file."
-             "For example: conn_query_str=ActualQPS==1000, qps_query_str=NumThreads==16."
+             "For example: conn_query_str=ActualQPS==1000, qps_query_str=NumThreads==16.",
```

Comment:
not sure what that example is but if ActualQPS is from fortio json, it's very unlikely to be a nice exact round number - if you ask for 1000 you're likely to get 999.97 or some such

Reply:
I think my formatter might have caught this. Having the hard equals isn't great, I agree. But it gets rounded to an int, so if you run it for long enough, the actual qps ends up being the desired qps (most of the time).